accounts/models.py

```diff
@@ -1,2 +1,30 @@
-# from django.db import models
-# from django.contrib.auth.models import User
+from django.db import models
+from django.contrib.auth.models import User
+# Each user can add, modify or remove course from is list
+# Each course contains coures id(default by Django), name, credinitials and grade
+
+
+class Course (models.Model):
+    user = models.ManyToManyField(User, blank=True)
+    course_name = models.CharField(max_length=50)
+    credinitials = models.IntegerField(max_length=1)
+    grade = models.IntegerField(max_length=3)
+    SEMESTER_NAME = (
```

Comment:
Put that at the top, not in the middle of the fields

Reply:
Ok, I didnwt saw your comments beacuse i was trying to find why the flake8 failed. I will change it.